Conventional/KnownPaths.cs

```diff
@@ -0,0 +1,12 @@
+using System.IO;
+
+namespace Conventional
+{
+    public static class KnownPaths
+    {
+        public static string SolutionRoot
+        {
+            get { return Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(), @"../../../")); }

```

Comment:
Probably not @bendetat - it is making a fairly broad assumption about your folder structure. I'll make it replaceable so that it can be set to whatever 'root' is applicable for your given solution.